pages/roles/index.tsx

```diff
@@ -0,0 +1,32 @@
+import useSWR from 'swr';
+import { Spin, Table, Typography } from 'antd';
+import CreateRoleForm from '../../components/forms/CreateRoleForm';
+import { ROLES_URL } from '../../api/rolesApi';
+
+const { Title } = Typography;
+
+function Applications() {
+  const { data } = useSWR(ROLES_URL);
```

Comment:
Can we have it as ROLES_API_ENDPOINT to be more descriptive? Super nit only, feel free to ignore.